README.md

```diff
@@ -35,10 +35,10 @@ To create a new Python project with this template:
 
 1. Install the latest [Cruft](https://github.com/cruft/cruft) and [Cookiecutter](https://github.com/cookiecutter/cookiecutter) in your [Python environment](https://github.com/pyenv/pyenv-virtualenv) with:
    ```sh
-   pip install --upgrade cruft>=2.12.0 cookiecutter>=2.1.1
+   pip install --upgrade "cruft>=2.12.0" "cookiecutter>=2.1.1"
    ```
-2. Create a new repository and clone it locally.
-3. In the directory that contains the cloned repository, run:
+2. Create a new (empty) repository and clone it locally.
```

Comment:
Does the empty matter here? As a reader, it leaves me wondering why that would be important.

Reply:
The first time I used the template, I thought step 2 referred to the cookie-cutter repo itself. (Also because it wasn't clear directly that the template is applied in step 3, but this part is resolved now.)
I would suggest to make this explicit, but adding `empty` is maybe not clear enough.
Another suggestion: `Create your new repository, then clone it locally`, where 'your' makes it clear it's not about the current cookie-cutter repo.

Reply:
To my mind, the 'your' refers to something that isn't defined yet. To solve that, how about:
> Create a new repository for your Python project, then clone it locally.